cfme/tests/infrastructure/test_host.py

```diff
@@ -331,3 +334,59 @@ def test_hosts_not_displayed_several_times(appliance, provider, setup_provider):
     provider.wait_for_delete()
     provider.create()
     assert host_count == navigate_to(appliance.collections.hosts, "All").paginator.items_amount
+
+
+@pytest.fixture
+def setup_provider_min_hosts(request, provider, min_hosts=2):
+    num_hosts = len(provider.data.get('hosts', {}))
+    if num_hosts < min_hosts:
+        pytest.skip(f'Not enough hosts({num_hosts}) to run test. Need at least {min_hosts}')
+    """Function-scoped fixture to set up a provider"""
+    return setup_or_skip(request, provider)
+
+
+@test_requirements.infra_hosts
+def test_infrastructure_hosts_refresh_multi(appliance, setup_provider_min_hosts, provider):
+    """
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/6h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Providers view.
+            2. Click on a provider guadicon, and then the hosts link along the top row of the view.
+            3. Select all hosts (need at least 2 hosts) by checking the box in upper left of
+               quadicons.
+            4. Click "Refresh Relationships and Power States" under the Configuration
+               dropdowm, and then click "OK" when prompted.
+        expectedResults:
+            1. Providers view is displayed.
+            2. Hosts view is displayed.
+            3.
+            4. "Refresh initiated for X Hosts from the CFME Database" is displayed in green
+               banner where "X" is the number of selected hosts. Properties for each host are
+               refreshed. Making changes to test pre-commithooks
+    """
+    hosts_view = navigate_to(provider.collections.hosts, "All")
+    num_hosts = len(hosts_view.entities.get_all())
+    if num_hosts < 2:
+        pytest.skip('not enough hosts in appliance UI to run test')
```

Comment:
This is duplicating the skip in the fixture above

Reply:
Hmm looking at Mike's comment above, I see there is a second assertion going on here.